components/search_results_header/search_results_header.tsx

```diff
@@ -65,7 +64,7 @@ export default class SearchResultsHeader extends React.PureComponent {
                                 id='rhs_header.expandSidebarTooltip.icon'
                                 defaultMessage='Expand Sidebar Icon'
                             >
-                                {(ariaLabel) => (
+                                {(ariaLabel: string | undefined) => (
```

Comment:
this can be written as optional property `ariaLabel?: string` ... ts-compiler will infer the `undefined` from it

Reply:
Thanks, I'll add these changes in